mosec/args.py

```diff
@@ -145,8 +153,17 @@ def parse_arguments() -> argparse.Namespace:
     return args
 
 
-def is_debug_mode() -> bool:
+def get_log_level() -> int:
     """Check if the service is running in debug mode."""
     parser = build_arguments_parser()
     args, _ = parser.parse_known_args(namespace=get_env_namespace())
-    return args.debug
+    if args.debug:
+        return logging.DEBUG
+
+    level_map = {
```

Comment:
We can do `getattr(logging, args.log_level.upper())` without this map.

Reply:
I have tried:

1. `logging._nameToLevel`: got a lint warning since it's private
2. `logging.getLevelName`: private, also the comments make me feel it's unsafe to use this one. BTW this func is also used by LogRecord.

`getattr(logging)` will search for lots of variables in this module, it's currently correct but I feel it's hard to control this behavior.